packages/cdk8s-cli/templates/java-app/.hooks.sscaff.js

```diff
@@ -35,7 +33,7 @@ exports.post = options => {
   }
 
   execSync(`mvn install`);
-  execSync(`${cli} import k8s -l java`);
+  execSync(`cdk8s import k8s -l java`);
```

Comment:
This will search the path instead of use the _current_ CLI. What's the rationale?

Reply:
On Windows it has trouble resolving `../../bin/cdk8s` so I switched to `cdk8s` from the path. These templates are used during the `cdk8s init ...` command and I'm making the assumption most users have it in their path rather than like `/usr/local/bin/cdk8s init ...`